attributes/config.rb

```diff
@@ -27,3 +27,21 @@
 # Set Hadoop attributes
 override['hadoop']['distribution'] = 'cdh'
 override['hadoop']['distribution_version'] = 5
+
+# Impala /etc/default/impala startup configurations
+default['impala']['config']['IMPALA_CATALOG_SERVICE_HOST'] = '127.0.0.1'
+default['impala']['config']['IMPALA_STATE_STORE_HOST'] = '127.0.0.1'
+default['impala']['config']['IMPALA_STATE_STORE_PORT'] = '24000'
+default['impala']['config']['IMPALA_BACKEND_PORT'] = '22000'
+default['impala']['config']['IMPALA_LOG_DIR'] = '/var/log/impala'
+default['impala']['config']['IMPALA_CATALOG_ARGS'] = ' -log_dir=${IMPALA_LOG_DIR} '
```

Comment:
Which trailing space?  I am not seeing a trailing space after any "\".
To which space are you referring?